pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -230,7 +230,7 @@ func init() {
 				},
 				{
 					APIGroups: []string{""},
-					Verbs:     sets.NewString("get", "list", "watch"),
+					Verbs:     sets.NewString("get", "list", "watch", "update"),
```

Comment:
Do you think that a label or relabel subresource on the replica set that would make the replica set controller relabel itself and then the pods would make any sense? It's not possible now so we need to handle it in deployments.